style.css

```diff
@@ -0,0 +1,1117 @@
+  font-weight: 700;
+  line-height: 1.25;
+}
+
+.rubric-thumbnail-bottom {
+  width: 100%;
+  margin-top: 1em;
+}
+
+.rubric-ul {
+  padding-left: 1em;
+}
+
+.rubric-ul > li {
+  padding-top: 0.5em;
+  padding-bottom: 0.5em;
+  border-bottom: 1px solid #eee;
+}
+
+.rubric-ul ::marker {
+  font-size: 0.7em;
+  color: #f42;
+}
+
+.rubric-bottom-list {
+  font-size: 0.85rem;
+  line-height: 1.5em;
+  font-weight: 400;
+  color: rgb(85, 85, 85);
+  padding-left: 0.5em;
+  font-family: ARS Maquette, Helvetica Neue, Arial, sans-serif;
+}
+
+.rubric-bottom-list :visited {
+  color: #aaa;
+}
+
+.rubric-bottom-list:hover,
+.rubric-bottom-list:focus {
+  color: black;
+  font-weight: 500;
+}
+
+/* ----styling latest deals--- */
+
+.latest-deals-thumbnails-container {
+  width: 100%;
+}
+
+.latest-deals-background-container {
+  position: relative;
+}
+
+.latest-deals-ribbon {
+  width: 30%;
+  position: absolute;
+  top: 0;
+  right: 0;
+}
+
+.latest-deals-background {
+  position: relative;
+  width: 100%;
+}
+
+.latest-deals-thumbnails-img-text {
+  transform: rotate(45deg);
+  font-weight: 800;
+  z-index: 3;
+  position: absolute;
+  top: 11%;
+  right: 0%;
+  font-size: 1vw;
+}
+
+.latest-deals-thumbnails-h3 {
+  color: black;
+  padding: 0.7em 0;
+  font-size: 1rem;
+  font-weight: bold;
+}
+
+.latest-deals-thumbnails-h4 {
+  font-size: 1.1rem;
+  color: #555;
+  font-weight: 1;
+}
+
+.latest-deals-thumbnails-h4-span {
+  text-decoration: line-through;
+  opacity: 0.7;
+  font-size: 0.8rem;
+}
+
+/* ----styling footer--- */
+
+.footer-bottom-nav {
+  background-color: #131313;
+  height: 65%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.footer-bottom-nav-icon-container {
+  display: flex;
+  list-style: none;
+}
+
+.footer-bottom-nav-icon {
+  color: white;
+  font-size: 1.3rem;
+}
+
+.footer-bottom-nav-icon-container li {
+  margin: 0 0.5em;
+  border-radius: 7rem;
+  background-color: rgba(36, 35, 35, 0.6);
+  padding: 0.8rem;
+  width: 3rem;
+  height: 3rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.footer-bottom-nav-links-container {
+  display: flex;
+  list-style: none;
+}
+
+.footer-bottom-nav-links-container li {
+  margin: 0 0.5em;
+}
+
+.footer-bottom-nav-icon-container li:hover,
+.footer-bottom-nav-icon-container li:focus {
+  border: solid 0.5px gray;
+  background-color: black;
+}
+
+.footer-bottom-nav-link:hover,
+.footer-bottom-nav-link:focus {
+  color: lightgray;
+}
+
+.foot-note {
+  background-color: black;
+  height: 35%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.foot-note-p {
+  width: 40%;
+  text-align: center;
+  font-size: 0.8rem;
+  line-height: 1.5em;
+}
+
+.foot-logo {
+  display: inline-block;
+  width: 2.6rem;
+}
+
+.foot-logo-stackpath {
+  width: 4rem;
+}
+
+#heart-icon {
+  color: red;
+}
+
+@media (max-width: 980px) {
+  header {
+    display: grid;
```

Comment:
- [ ] This media query is giving some issues, please check that is the correct `max-width`